### PR TITLE
Feature 12465 display a history of sent external emails

### LINKED ISSUE #12465: Display a history of sent external emails [3]
### Feature Description

External emails that have already been sent successfully must be clearly presented to the user.

Linked to #12402 

### Added Value/Benefit

Traceability and clarity.

### Acceptance Criteria

- [x] The `ManualMessageLog` entity and DTO have been extended with new fields:
  - String `usedTemplate`
  - Set<String> `attachedDocuments`
  - String `emailAddress`
  - reference fields for caze, contact, event participant, and travel entry
- [x] When sending out an external email, a log entry of type `EMAIL` is added to the database, with all existing and new fields filled in (only one reference of course, the one of the entity from which the email has been sent); the `attachedDocuments` field contains the file names of all documents attached to the email, and the email address field contains the email address the email has been sent to
- [x] In the side component introduced in #12463, as soon as at least one mail has been sent to the person through the displayed entity, a list of log entries is displayed, listing all log entries of the type `EMAIL` for the respective person and the referenced entity where `usedTemplate` is not empty
- [x] The following fields are displayed in the side component for each log entry, underneath each other on the left side:
  - Date sent
  - Used template
  - Sending user
- [x] In addition, an info button is displayed on the right side (vertically centered); when clicked on, it brings up a pop-up that contains additional information about the sent email:
  - Email address the email has been sent to
  - Attached document names

### Additional Information

### PR DESCRIPTION
**Pull request by leventegal-she**
Fixes #12465

### CODE REVIEW

File: sormas-api/src/main/java/de/symeda/sormas/api/i18n/Strings.java [hunk 1]
```diff
@@ -603,6 +603,7 @@ public interface Strings {
 	String headingExposureInvestigation = "headingExposureInvestigation";
 	String headingExtendFollowUp = "headingExtendFollowUp";
 	String headingExtendQuarantine = "headingExtendQuarantine";
+	String headingExteranlEmailDetails = "headingExteranlEmailDetails";
```
Comment (carina29) [inline sormas-api/src/main/java/de/symeda/sormas/api/i18n/Strings.java:606]: Typo

File: sormas-api/src/main/java/de/symeda/sormas/api/document/DocumentFacade.java [hunk 2]
```diff
@@ -35,7 +35,7 @@ public interface DocumentFacade {
 
 	List<DocumentDto> getDocumentsRelatedToEntity(DocumentRelatedEntityType type, String uuid);
 
-	List<DocumentReferenceDto> getReferencesRelatedToEntity(DocumentRelatedEntityType type, String uuid, Set<String> fileExtensions);
+    List<DocumentReferenceDto> getReferencesRelatedToEntity(DocumentRelatedEntityType type, String uuid, Set<String> fileExtensions);
```
Comment (carina29) [inline sormas-api/src/main/java/de/symeda/sormas/api/document/DocumentFacade.java:38]: Code formatting

File: sormas-ui/src/test/resources/META-INF/persistence.xml [hunk 3]
```diff
@@ -62,7 +62,7 @@
 		<class>de.symeda.sormas.backend.sormastosormas.share.outgoing.ShareRequestInfo</class>
 		<class>de.symeda.sormas.backend.document.Document</class>
 		<class>de.symeda.sormas.backend.exposure.Exposure</class>
-		<class>de.symeda.sormas.backend.common.messaging.ManualMessageLog</class>
+        <class>de.symeda.sormas.backend.manualmessagelog.ManualMessageLog</class>
```
Comment (carina29) [inline sormas-ui/src/test/resources/META-INF/persistence.xml:65]: Code formatting

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=12830

**Review by carina29 — APPROVED** [review]

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=12830

**Pull request merged**